rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -95,37 +98,45 @@ private boolean shouldUseRskPublicKey() {
 
     private void logAddSignatureInSolidityFormat(byte[] rskTxHash, String federatorRskAddress, BtcECKey federatorPublicKey) {
         CallTransaction.Function event = BridgeEvents.ADD_SIGNATURE.getEvent();
-        byte[][] encodedTopicsInBytes = event.encodeEventTopics(rskTxHash, federatorRskAddress);
-        List<DataWord> encodedTopics = LogInfo.byteArrayToList(encodedTopicsInBytes);
-        byte[] encodedData = event.encodeEventData(federatorPublicKey.getPubKey());
 
-        this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, encodedTopics, encodedData));
+        byte[][] encodedTopicsSerialized = event.encodeEventTopics(rskTxHash, federatorRskAddress);
+        List<DataWord> encodedTopics = getEncodedTopics(encodedTopicsSerialized);
+
+        byte[] federatorPublicKeySerialized = federatorPublicKey.getPubKey();
+        byte[] encodedData = event.encodeEventData(federatorPublicKeySerialized);
+
+        addLog(encodedTopics, encodedData);
     }
 
     @Override
     public void logReleaseBtc(BtcTransaction btcTx, byte[] rskTxHash) {
         CallTransaction.Function event = BridgeEvents.RELEASE_BTC.getEvent();
-        byte[][] encodedTopicsInBytes = event.encodeEventTopics(rskTxHash);
-        List<DataWord> encodedTopics = LogInfo.byteArrayToList(encodedTopicsInBytes);
-        byte[] encodedData = event.encodeEventData(btcTx.bitcoinSerialize());
 
-        this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, encodedTopics, encodedData));
+        byte[][] encodedTopicsSerialized = event.encodeEventTopics(rskTxHash);
+        List<DataWord> encodedTopics = getEncodedTopics(encodedTopicsSerialized);
+
+        byte[] rawBtcTx = btcTx.bitcoinSerialize();
```

Comment:
```suggestion
        byte[] btcTxSerielized = btcTx.bitcoinSerialize();
```

Reply:
i like better the name `rawBtcTx`, since it is the usual way we call the serialized tx

Reply:
Yes, but it is called raw transaction when it is in hex string. See the [json rpc API methods](https://developer.bitcoin.org/reference/rpc/decoderawtransaction.html) what operates on raw tx inputs

In this case, where you are handling bytes, I would call it `...Serialized`

Reply:
good point, updated :)